packages/xrpl/src/models/ledger/Offer.ts

```diff
@@ -34,9 +47,20 @@ export default interface Offer extends BaseLedgerEntry, HasPreviousTxnID {
   OwnerNode: string
   /** The time this Offer expires, in seconds since the Ripple Epoch. */
   Expiration?: number
+
+  /** The domain that the offer must be a part of. */
+  DomainID?: string
+
+  /**
+   * An additional list of order book directories that this offer belongs to.
+   * Currently this field only applicable to hybrid offers.
+   */
+  AdditionalBooks?: Book[]
 }
 
 export enum OfferFlags {
   lsfPassive = 0x00010000,
   lsfSell = 0x00020000,
+  // Indicates the offer is hybrid. (meaning it is part of both a domain and open order book)
+  lsfHybrid = 0x00040000,
```

Comment:
```suggestion
  lsfHybrid = 0x00040000,
```
Should probably remove the comment since this can be looked up on docs. The other flags don't have it too.